frontend/server/src/DAO/CoderOfTheMonth.php

```diff
@@ -297,7 +297,7 @@ public static function calculateCoderOfTheMonth(
                 WHERE
                     `time` = ?;';
 
-        /** @var list<array{coder_of_the_month_id: int, description: null|string, interview_url: null|string, rank: int, school_id: int|null, selected_by: int|null, time: string, user_id: int}> */
+        /** @var list<array{ category: string, coder_of_the_month_id: int, description: null|string, interview_url: null|string, rank: int, school_id: int|null, selected_by: int|null, time: string, user_id: int}> */
```

Comment:
```suggestion
        /** @var list<array{category: string, coder_of_the_month_id: int, description: null|string, interview_url: null|string, rank: int, school_id: int|null, selected_by: int|null, time: string, user_id: int}> */
```

Casi, sólo te sobra un espacio.